docs/pages/router/reference/url-parameters.mdx

```diff
@@ -358,3 +358,12 @@ export default function User() {
   );
 }
 ```
+
+## Reserved parameters
+
+Certain URL parameters are reserved for internal use by Expo Router and React Navigation. Avoid using these names for your own parameters to prevent conflicts:
```

Comment:
```suggestion
Certain URL parameters are reserved for internal use by Expo Router and React Navigation. Avoid using the following names for your own parameters to prevent conflicts:
```